utils/dwarf.c

```diff
@@ -907,6 +907,25 @@ static void get_source_location(Dwarf_Die *die, struct build_data *bd,
 	if (dfile == NULL)
 		return;
 
+	Dwarf_Die cudie;
+	dwarf_diecu(die, &cudie, NULL, NULL);
```

Comment:
I remember the coding style of uftrace about declaration you said before. 
but I followed style of previous code of this function. give me any opinion.

Reply:
I still prefer declaration at the beginning but there are some places violate it.  Please do not add more.